### PR TITLE
refinements

### PR DESCRIPTION
**Pull request by andrewferlitsch**

### CODE REVIEW

File: notebooks/templates/ai_platform_notebooks_template.ipynb [hunk 1]
```diff
@@ -1,524 +1,634 @@
 {
```
Comment (kweinmeister) [inline notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>Is there a missing tick for markdown around the ###?</p><p> </p><p>Also, is something missing around "1. into these commands?"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (kweinmeister) [inline notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>Turn this into a complete sentence? And it should say "may be able".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (kweinmeister) [inline notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>Is there a spacing issue around following the guidelines?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (kweinmeister) [inline notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>All of the examples are TF-specific (which is fine), although template users may be using other frameworks. Should we say something like: "for example, when using TensorFlow:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (kweinmeister) [inline notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>Use markdown around the pip command</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (andrewferlitsch) [reply notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>That's an existing cell. I made no changes. Not sure what you mean by extra tick around ###. I do see an extra period in 1. -- so I remove it.</p>

Comment (andrewferlitsch) [reply notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>done</p>

Comment (andrewferlitsch) [reply notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>Existing cell, I made no changes -- can't see the spacing issue?</p>

Comment (andrewferlitsch) [reply notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>Existing cell. I added your suggestion.</p>

Comment (andrewferlitsch) [reply notebooks/templates/ai_platform_notebooks_template.ipynb:1]: <p>existing cell. added your suggestion.</p>

**Pull request merged**